sympy/solvers/solvers.py

```diff
@@ -375,6 +375,25 @@ def checksol(f, symbol, sol=None, **flags):
     # returns None if it can't conclude
     # TODO: improve solution testing
 
+def is_eq_bool(eq):
+    """Returns true if Eq or Ne has True or False as an argument"""
+    if isinstance(eq, (Equality, Unequality)):
+        return any(side in (S.false, S.true) for side in eq.args)
+
+def invert_eq_bool(fi):
```

Comment:
These functions should be private so their names should begin with an underscore e.g. `_invert_eq_bool`.